src/main/java/com/thevoxelbox/voxelsniper/brush/type/stencil/StencilBrush.java

```diff
@@ -49,7 +51,7 @@
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
-        String firstParameter = parameters[1];
+        String firstParameter = parameters[0];
         byte pasteParam;
         if (firstParameter.equalsIgnoreCase("info")) {
             messenger.sendMessage(ChatColor.GOLD + "Stencil brush Parameters:");
```

Comment:
One line below, `b schem`. That arg is invalid, the brush doesn't register itself under that name.
Actually, looking at the way args are sliced, using the command without input `/b stencil replace` causes an AIOOBE.